data-structures-queues/is-empty.js

```diff
@@ -0,0 +1,8 @@
+/* exported isEmpty */
+
+function isEmpty(queue) {
+  if (queue.peek() === undefined) {
+    return true;
+  }
+  if (queue.peek() >= 0) { return false; }
```

Comment:
Here you can just return `false`, no need for an `if` here, the first `if` statement is all that is needed.